integration-tests/sceptre-project/templates/loggroup14.yaml

```diff
@@ -0,0 +1,6 @@
+Resources:
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: IntegrationTest14
```

Comment:
I'd recommend not using a specific LogGroupName. It's optional in CloudFormation. If you don't specify it, CF will create a unique name for you automatically. In these integration tests, if they ever fail to be torn down somehow, I don't want all future tests to fail because the log group name already exists.

Anyway, I'd recommend just omitting this one.

Reply:
@jfalkenstein I need to name the log group so as to be able to then later change it. Otherwise I need much more complicated (and slower) code to find the auto-generated log group name by looking at all resources by stack name.

Reply:
See comment below. https://github.com/Sceptre/sceptre/pull/1170#discussion_r776158908